config/schemas/templates/page.json

```diff
@@ -3,15 +3,15 @@
   "previewUrl": "",
   "request": [
     {
-      "url": "/api/pagebuilder/search?storeId=B2B-store&type=themes&query=*.page",
+      "url": "/api/pagebuilder/search?storeId=B2B-store&type=pages&query=*.page",
       "method": "get",
       "cacheable": true,
       "response": {
-        "selector": "Object.keys(this.response).map(x => ({ ...this.item, ...this.response[x], request: undefined, path: this.response[x].path.substring('/default'.length) })).reduce((acc, x) => ({ ...acc, [x.name]: x }), {})",
+        "selector": "Object.keys(this.response).map(x => ({ ...this.item, ...this.response[x], request: undefined })).reduce((acc, x) => ({ ...acc, [x.name]: x }), {})",
```

Comment:
This code raises doubts about its location here

Reply:
no doubt! its location must be there

Reply:
Please talk to us about how it works later.